packages/@glimmer/integration-tests/test/initial-render-test.ts

```diff
@@ -29,6 +29,11 @@ import {
 import { expect } from '@glimmer/util';
 import { SimpleElement } from '@simple-dom/interface';
 
+// `window.ActiveXObject` is "falsey" in IE11 (but not `undefined` or `false`)
+// `"ActiveXObject" in window` returns `true` in all IE versions
+// only IE11 will pass _both_ of these conditions
+const isIE11 = !(window as any).ActiveXObject && 'ActiveXObject' in window;
```

Comment:
It would be good to annotate this stuff so it's easy to cleanup in a #worldwithoutIE11.

Reply:
Happy to! How do you think I should annotate?